data/saas/config/aircall_config.yml

```diff
@@ -0,0 +1,109 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Aircall SaaS Config
+  type: aircall
+  description: A sample schema representing the Aircall connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: api.aircall.io
+    - name: api_token
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: api_key
+      configuration:
+        headers:
+          - name: Authorization
+            value: Basic <api_token>
+
+  test_request:
+    method: GET
+    path: /v1/contacts/
+    param_values:
+      - name: api_token
+        connector_param: api_token
```

Comment:
remove api_token from all calls now that it is in the client_config

Reply:
api_token removed from all end points